skops/io/_persist.py

```diff
@@ -64,8 +66,19 @@ def dump(obj: Any, file: str | Path | BinaryIO) -> None:
         convention, we recommend to use the ".skops" file extension, e.g.
         ``save(model, "my-model.skops")``.
 
+    compression: int, default=zipfile.ZIP_STORED
+        The compression method to use. See :class:`zipfile.ZipFile` for more
+        information.
+
+        .. versionadded:: 0.7
+
+    compresslevel: int, default=None
+        The compression level to use. See :class:`zipfile.ZipFile` for more
+        information.
```

Comment:
Do we want to make a recommendation (here or in the docs) about what settings to use? E.g. "if file size is no issue, use the defaults, as they are the fastest. Otherwise, use <setting that works on well>".

Reply:
added something in the docs.